eland/ml/pytorch/transformers.py

```diff
@@ -732,7 +732,7 @@ def _create_config(
             else:
                 sample_embedding = self._traceable_model.sample_output()
                 if type(sample_embedding) is tuple:
-                    text_embedding, _ = sample_embedding
+                    text_embedding, *_ = sample_embedding
```

Comment:
This works but it's maybe obscure Python code. We don't actually need to unpack the whole tuple, so it might make it clearer to just extract the first element, which is what we actually care about:

```python
text_embedding = sample_emebdding[0]
```